### PR TITLE
Register sxi.is-a.dev

### PR DESCRIPTION
**Pull request by Sxinar**
Register sxi.is-a.dev with CNAME record pointing to sxinar.github.io.

### CODE REVIEW

**Comment by Sxinar** [comment]
@CuteDog5695

File: domains/sxi.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Sxinar",
+           "email": "130384567+Sxinar@users.noreply.github.com",
```
Comment (Stef-00012) [inline domains/sxi.json:4]: You can't use a noreply email

**Review by Stef-DP — DISMISSED** [review]

**Review by iostpa — CHANGES_REQUESTED** [review]
What Stef said

File: domains/sxi.json (hunk 1 above)
Comment (21Z) [reply domains/sxi.json:4]: If you don't want your email to be exposed, you can remove the remove the email, but your discord should be there.

Comment (Sxinar) [reply domains/sxi.json:4]: @Stef-00012 oshidev@proton.me

Comment (CuteDog5695) [reply domains/sxi.json:4]: you need to edit the json file and replace the email with that by the way

Comment (CuteDog5695) [reply domains/sxi.json:4]: > you need to edit the json file and replace the email with that btw

Comment (Sxinar) [reply domains/sxi.json:4]: @CuteDog5695 update now.

Comment (CuteDog5695) [reply domains/sxi.json:4]: why cant you?

Comment (Sxinar) [reply domains/sxi.json:4]: > neden yapamıyorsun?

I am Turkish and I updated the necessary file, only my English is poor.

Comment (CuteDog5695) [reply domains/sxi.json:4]: fine, ill update the file for you

**Review by iostpa — APPROVED** [review]
Should be fine now

**Pull request merged**